extra/3-magic-8-ball.js

```diff
@@ -45,10 +93,38 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+var answers = [
+	"It is certain.",
+	"It is decidedly so.",
+	"Without a doubt.",
+	"Yes - definitely.",
+	"You may rely on it.",
+	"As I see it, yes.",
+	"Most likely.",
+	"Outlook good.",
+	"Yes.",
+	"Signs point to yes.",
+	"Reply hazy, try again.",
+	"Ask again later.",
+	"Better not tell you now.",
+	"Cannot predict now.",
+	"Concentrate and ask again.",
+	"Don't count on it.",
+	"My reply is no.",
+	"My sources say no.",
+	"Outlook not so good.",
+	"Very doubtful.",
+];
+
 function shakeBall() {
-  //Write your code in here
+	//Write your code in here
+
+	console.log("The ball has shaken!");
+	return answers[Math.floor(Math.random() * 20)];
```

Comment:
You could use answers.length here instead of 20, then you won't need to update the number if you add or remove anything from the array